### ISSUE TITLE
Could not install package 'ServiceStack.Common 4.5.14'. You are trying to install this package into a project that targets 'MonoAndroid,Version=v8.0', but the package does not contain any assembly references or content files that are compatible with that framework.

### ISSUE THREAD

**Issue by whitedevils** [ServiceStack/Issues#578]
Hi, i have tryed to install servicestack package to my xamarin android project but i get this error.
Can anyone help me please?
Thanks

**Comment by mythz** [comment]
The error message indicates you're trying to install the ServiceStack.Common server library in an Android project but only the `ServiceStack.Client` NuGet package can be used in iOS/Android.

**Comment by whitedevils** [comment]
Yes but if i have to use servicestack Authentication feature to authenticate my client, i have to install all servicestack package. right?

**Comment by mythz** [comment]
No, the `Authenticate` DTO is in the ServiceStack.Client package.

**Comment by whitedevils** [comment]
ok, maybe i've lost some info, let me try again. Thanks for reply.

**Comment by whitedevils** [comment]
Ok, i have tryed but now when i initialize Jsonservicecliente i recieved this error:
"System.TypeInitializationException: An exception was thrown by the type initializer for ServiceStack.ServiceClientBase"

**Comment by mythz** [comment]
Please provide the full StackTrace and the client source code you're using.

**Comment by whitedevils** [comment]
I found the solution! IosPclExportClient.Configure() and  AndroidPclExportClient.Configure() must be added in startup class of xamarin platform projects!
